AdyenComponents/Boleto/BoletoComponent.swift

```diff
@@ -0,0 +1,233 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+import UIKit
+
+/// A component that provides a form for Boleto payment.
+public final class BoletoComponent: PaymentComponent, LoadingComponent, PresentableComponent, Localizable, Observer {
```

Comment:
please pull from develop, because of the gift card flow, every component need to have an `order` to be able to participate in partial payments.